db/migrate/20151220030058_add_estimated_dob_to_students.rb

```diff
@@ -0,0 +1,7 @@
+class AddEstimatedDobToStudents < ActiveRecord::Migration
+  def change
+    add_column :students, :estimated_dob, :boolean
+    change_column :students, :estimated_dob, :boolean, null: false #This is because of an SQLite bug which won't lett us add a column with a NOT NULL constraint
```

Comment:
OK. I thought it would be a default if you haven't passed a value in INSERT, but what you're saying is a default for already existing rows in a table.

Anyway, I updated it so it would be super cool if you could take another look :)